styles.css

```diff
@@ -60,6 +60,53 @@ body {
   font: 18px/21px Roboto, arial, helvetica, sans-serif;
 }
 
+@keyframes pulse1 {
+  0% {
+    transform: scale(.9, .9);
+  }
+  30% {
+    transform: scale(1, 1); opacity: .8;
+    }
+  50% {
+    transform: scale(1.2, 1.2); opacity: .7;
+  }
+  70% {
+    transform: scale(1.3, 1.3); opacity: .5;
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6357730)